bokehjs/src/lib/models/tools/inspectors/hover_tool.ts

```diff
@@ -70,7 +70,12 @@ export class HoverToolView extends InspectToolView {
 
   initialize(): void {
     super.initialize()
+    this._clear_ttfields()
+  }
+
+  protected  _clear_ttfields(): void {
     this._ttmodels = null
+    delete this._template_el
     this._ttviews = new Map()
```

Comment:
`_ttviews` must not be reinitialized, otherwise `build_views()` on line 127 will fail to do a clean up.

Reply:
For the sake of consistency change the type of `_template_el` to `| null` (instead of `?`) and use `null`.